scripts/lib.sh

```diff
@@ -27,6 +27,10 @@ function await_block_sync() {
   echo "Mint at ${EXPECTED_BLOCK_HEIGHT}H"
 }
 
+function await_all_peers() {
```

Comment:
Does this await *all* peers or *enough to form a consensus* clients?

Reply:
The new `api` command in the CLI waits for all peers, which allowed me to refactor this script